RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Refrigerator :IComparable<Refrigerator>
+    {
+        private string _model;
+
+
+        public Guid id { get; }
+        public string Model
+        {
+            get
+            {
+                return _model;
+            }
+            private set
+            {
+                if (!value.Equals(""))
+                    _model = value;
+                else
+                    throw new ArithmeticException("Illegal model ,model must contain a String ");
+            }
+        }
+        public string Color { get; set; }
+        public int NumberOfShelfs { get; set; }
+        public List<Shelf> Shelves;
+
+
+        public Refrigerator(string model, string color, int shelf)
+        {
+            id = Guid.NewGuid();
+            try
+            {
+                Model = model;
+
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message); 
+            }
+            Color = color;
+            NumberOfShelfs = shelf;
+            Shelves = new List<Shelf>();
+            this.AddShelfs();
+        }
+        public void AddShelfs()
+        {
+            while (this.Shelves.Count() < this.NumberOfShelfs)
+            {
+                this.Shelves.Add(new Shelf(this.Shelves.Count()));
+            }
+        }
+        public override string ToString()
+        {
+            string shelfs = "";
+            foreach (Shelf shelf in Shelves)
+            {
+                shelfs += "\n=================================================================================\n";
+                shelfs += "\n" + shelf.ToString();
+            }
+            return "Refrigerator id: " + id + " Model: " + Model + " Color: " + Color + " with " + NumberOfShelfs + " shefls containes \n" + shelfs;
+        }
+        public int SpaceInRefrigerator()
+        {
+            int spaceSum = 0;
+            foreach (Shelf shelf in Shelves)
+                spaceSum += shelf.Space;
+            return spaceSum;
+        }
+        public bool AddItem(Item item)
+        {
+            int count = 0;
+            foreach (Shelf shelf in Shelves)
+            {
+                if (shelf.IsSpaceInShelf(item.Space))
+                {
+                    shelf.AddItem(item);
+                    return true;
+                }
+                else
+                {
+                    if (item.Space > shelf.ShelfSpace)
+                        count++;
+                }
+
+            }
+            if (count == this.Shelves.Count)
+                Console.WriteLine("There is no shelf with this amount of space this item is too big");
+            else
+                Console.WriteLine("There is no more space in Refrigeratir, you might want to clean refrigerator!");
+            return false;
+        }
+        public Item GetItem(string id)
```

Comment:
method is used just for removing the item so better return bool:

public bool RemoveItem(string id)
{
    foreach (var shelf in Shelves)
    {
        var item = shelf.GetItem(id);
        if (item != null)
        {
            shelf.RemoveItem(item);
            return true;
        }
    }

    Console.WriteLine("There is no item with this id ");
    return false;
}